contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -156,6 +156,32 @@ contract VaultHub is PausableUntilWithRoles {
         ) >= STETH.getPooledEthBySharesRoundUp(socket.sharesMinted);
     }
 
+    /// @notice provides a gap for rebalancing vault
+    /// @param _vault vault address
+    /// @return amount to rebalance
+    function rebalanceShortfall(address _vault) public view returns (uint256) {
```

Comment:
Must have a separate tests set to cover all path of the function (reverts, success)